src/annotator/plugin/bucket-bar.js

```diff
@@ -22,44 +26,68 @@ function scrollToClosest(anchors, direction) {
 export default class BucketBar extends Delegator {
   constructor(element, options, annotator) {
     const defaultOptions = {
-      // gapSize parameter is used by the clustering algorithm
-      // If an annotation is farther then this gapSize from the next bucket
-      // then that annotation will not be merged into the bucket
-      // TODO: This is not currently used; reassess
-      gapSize: 60,
-      html: '<div class="annotator-bucket-bar"></div>',
       // Selectors for the scrollable elements on the page
-      scrollables: ['body'],
+      scrollables: [],
```

Comment:
Scrutiny here, please: we have `scroll` handling on `window`. Is it necessary to have it on `body`, too, by default?